keras_cv/layers/object_detection/multi_class_non_max_suppression_test.py

```diff
@@ -12,30 +12,62 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import unittest
+
 import tensorflow as tf
 
 from keras_cv import layers as cv_layers
 
 
+def decode_predictions_output_shapes():
```

Comment:
any reason to make this a global helper function?  

Can you define it inside of the unit test just to keep the definition local to the test?  So define it in the test body, then compile it.

Reply:
Where do you want to move?
As It is shared between 3 test classes.

Reply:
ah ok - then lets just leave it.